### PR TITLE
Fixes broken link in password expiration article

### PR DESCRIPTION
**Pull request by robdy**
As in the title - the link is broken.

Relevant article: [Set the password expiration policy for your organization
](https://docs.microsoft.com/en-us/microsoft-365/admin/manage/set-password-expiration-policy?view=o365-worldwide)

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ description: "Learn how an admin can set a password expiration policy for your b
 
 ## Before you begin
 
-This article is for people who set password expiration policy for a business, school, or nonprofit. To complete these steps, you need to sign in with your Microsoft 365 admin account. [What's an admin account?](Overview of the Microsoft 365 admin center](../admin-overview/admin-center-overview.md).
+This article is for people who set password expiration policy for a business, school, or nonprofit. To complete these steps, you need to sign in with your Microsoft 365 admin account. [See Who is an admin to learn about admin accounts](../admin-overview/admin-center-overview.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/set-password-expiration-policy.md]:
```suggestion
This article is for people who set password expiration policy for a business, school, or nonprofit. To complete these steps, you need to sign in with your Microsoft 365 admin account. See [Who is an admin](../admin-overview/admin-center-overview.md) to learn about admin accounts.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@robdy Please implement the suggested changes. Thanks

**Comment by robdy** [comment]
@yogkumgit it's done now.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd @robdy Please resolve the conflict so that @kwekuako can merge it. thanks

**Comment by robdy** [comment]
@yogkumgit conflict resolved 👍

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ description: "Learn how an admin can set a password expiration policy for your b
 
 ## Before you begin
 
-This article is for people who set password expiration policy for a business, school, or nonprofit. To complete these steps, you need to sign in with your Microsoft 365 admin account. [What's an admin account?](/microsoft-365/admin/add-users/about-admin-roles).
+This article is for people who set password expiration policy for a business, school, or nonprofit. To complete these steps, you need to sign in with your Microsoft 365 admin account. See [Who is an admin](../admin-overview/admin-center-overview.md) to learn about admin accounts.
```
Comment (tiburd) [inline microsoft-365/admin/manage/set-password-expiration-policy.md:34]:
```suggestion
This article is for people who set password expiration policy for a business, school, or nonprofit. To complete these steps, you need to sign in with your Microsoft 365 admin account. To learn about admin accounts, see [Who is an admin](../admin-overview/admin-center-overview.md#who-is-an-admin).
```

**Comment by tiburd** [comment]
@kwekuako 
Hi, please review the submitted changes and my not-yet-committed edit. Commit and merge the PR if you agree with the changes.
Thanks!
UPDATE: Disregard. I see the other PR that you merged. We'll close this one.

**Comment by robdy** [comment]
@tiburd I think I mixed up something while resolving a conflict. Apologies for that!

**Comment by robdy** [comment]
Oh and btw #7324 is conflicting with this PR. I thought it's worth mentioning

**Comment by tiburd** [comment]
@robdy 
Because the article author approved and merged the other PR, I'm going to close this PR.
Thanks so much for your submission, though!
Best wishes!

**Pull request closed without merging**